config/config.exs

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+config :smartsheet, api_key: System.get_env("SMARTSHEET_API_KEY")
```

Comment:
This is going to run at compile time, so if the API key isn't set then this value will always be nil in the application.

Reply:
is this why you advocated for basically duping this call, when it's actually used? (I *think* that's what you meant/said) see https://github.com/fast-radius/smartsheet/pull/1#discussion_r457540809

Reply:
Yeah pretty much. I think you actually want to kill it in the config and just leave the call in the code with the `||`. That way a user of the library can set it in their own config or use the env var.

Reply:
made the changes you requested here https://github.com/fast-radius/smartsheet/pull/4/files and took the liberty of merging -- thanks for the review/suggestions